src/uct/ib/base/ib_md.c

```diff
@@ -31,6 +31,10 @@
 
 #define UCT_IB_MD_RCACHE_DEFAULT_ALIGN 16
 
+/* TODO: add invalidation support */
+#define UCT_IB_MD_MEM_DEREG_CHECK_PARAMS(_params) \
+    UCT_MD_MEM_DEREG_CHECK_PARAMS(_params, 0)
```

Comment:
do we really need this macro?

Reply:
I was going to extend it later.
removed for now